### ISSUE TITLE
ndcg

### ISSUE THREAD

**Issue by xxxxwx** [ywjawmw/SMGCN-torch-master#1]
Hello, may I ask why after running your reproduced code, the effect of ndcg as an evaluation indicator is much worse than the original paper, is there any problem somewhere?

**Comment by ywjawmw** [comment]
Yes, you are right.  We implement NDCG using its original setting, which should be implemented in the original code, so that the indicator is reasonable.

**Comment by xxxxwx** [comment]
Hello, I'm not sure if you have tried running the author's publicly available TensorFlow version of the source code. When running multiple times with the optimal parameters described in the author's paper, the SMGCN model's score is consistently 0.01 lower than mentioned in the paper, while the scores of the compared ngcf and gcmc models are 0.01 higher than those mentioned in the paper. Similar results are obtained after multiple runs. I'm wondering if there is an issue with my settings? What kind of results did you get?

**Comment by BossTJ** [comment]
> 您好，请问为什么运行您复制的代码后，ndcg作为评估指标的效果比原论文差很多，是不是哪里有问题？

您好，您有这个pretrain预训练这个文件吗？
